### ISSUE TITLE
mistral qlora fine tuning

### ISSUE THREAD

**Issue by gyuwon12** [axolotl-ai-cloud/axolotl#776]
### Please check that this issue hasn't been reported before.

- [X] I searched previous [Bug Reports](https://github.com/OpenAccess-AI-Collective/axolotl/labels/bug) didn't find any similar reports.

### Expected Behavior

https://github.com/OpenAccess-AI-Collective/axolotl/tree/main/examples/mistral

I'm fine tuning with the mistral-7B model, but there's a problem that the loss is really big. What's the reason? Another similar llama2 model(13b, 70b), a 2k test dataset in .yml, converges with a loss of 1.0 under, and under the same condition, the mistral is stamped with 5.xx.

### Current behaviour

A loss such as llama2 should be taken, but it outputs more than 5 times larger (with qlora)

### Steps to reproduce

I ran the file in the readme. Please check.

### Which Operating Systems are you using?

- [X] Linux
- [ ] macOS
- [ ] Windows

### Python Version

3.10

### axolotl branch-commit

x

### Acknowledgements

- [X] My issue title is concise, descriptive, and in title casing.
- [X] I have searched the existing issues to make sure this bug has not been reported yet.
- [X] I am using the latest version of axolotl.
- [X] I have provided enough information for the maintainers to reproduce and diagnose the issue.

**Comment by casper-hansen** [comment]
What is your config?

**Comment by winglian** [comment]
@gyuwon12 I ran the mistral/qlora.yml example and got sub-1.0 loss on the first step.

```

{'loss': 0.9864, 'learning_rate': 2e-05, 'epoch': 0.33}                                                                                                                                                                                                                                         
 33%|████████████████████████████████████████████████████████████████████████████████████▎                                                                                                                                                                        | 1/3 [00:54<01:48, 54.18s/it]
```

**Comment by gyuwon12** [comment]
@winglian Omg! Can you share your config?

**Comment by ehartford** [comment]
I find this is usually caused by a bad learning rate.

**Comment by winglian** [comment]
> @winglian Omg! Can you share your config?

It's simply the one in the examples. https://github.com/OpenAccess-AI-Collective/axolotl/blob/main/examples/mistral/qlora.yml
